discovery-provider/plugins/notifications/src/processNotifications/mappers/follow.ts

```diff
@@ -104,6 +110,8 @@ export class Follow extends BaseNotification<FollowNotificationRow> {
     ) {
       // TODO: Send out email
     }
+
+    await sendBrowserNotification(userNotificationSettings, this.receiverUserId, title, body)
```

Comment:
any reason why this goes before the mobile push notification call unlike the previous notifications where it sends browser first?

Reply:
i see a few of them have this order, could we keep it consistent across the mappers?

Reply:
Ah yes, I wrote them this way to start but then copying to each file was easier keeping them together. I'll fix the ones that are separated.